### DISCUSSION TITLE
Auto Role

### DISCUSSION THREAD

**Discussion by ghost** [Scythe-Anticheat/Scythe-Anticheat#93] [Help and Support]
Is there any way i can auto assign a [member] tag to all players with command blocks? i have tried /tag @a add “tag:Member” and that doesn’t seem to work, it only seems to work when i individual add players ranks using the scythe command, !tag {player} Member, which is much less efficient and won’t let me use the tags in command blocks for crates, staff permissions etc.
Any ideas on how i can fix this?

**Comment by RSChao** [comment]
/tag @a[hasitem="command_block"] add "tag:Member"

idk how hasitem arg works so may not do but hasitem is definitely the way to go there
